pkg/sdk/symbols/info/info.go

```diff
@@ -105,7 +107,27 @@ func plugin_get_required_api_version() *C.char {
 	return (*C.char)(pRequiredAPIVersion.CharPtr())
 }
 
+func string_split(version string) (string, string, string) {
```

Comment:
```suggestion
func splitVersionString(version string) (string, string, string) {
```
Naming and camel case.